tools/distpartitioning/dataset_utils.py

```diff
@@ -126,10 +128,13 @@ def get_dataset(input_dir, graph_name, rank, world_size, schema_map):
                     if not os.path.isabs(nfeat_file):
                         nfeat_file = os.path.join(input_dir, nfeat_file)
                     logging.info(f'Loading node feature[{feat_name}] of ntype[{ntype_name}] from {nfeat_file}')
-                    nfeat.append(np.load(nfeat_file))
+                    nfeat.append(
+                        array_readwriter.get_array_parser(
```

Comment:
could we have a wrapper that could help create `fmt_meta` for the caller? such as `utils.read(nfeat_file)`?